python/device/device.py

```diff
@@ -162,10 +191,12 @@ def __init__(self):
         self.pingback_wait_list = {}
         self.incoming_pingback_response_queue = queue.Queue()
         # for telemetry
-        self.outgoing_send_message_queue = queue.Queue()
+        self.outgoing_test_message_queue = queue.Queue()
```

Comment:
What is a `test message`?

Reply:
We can talk about this when we meet.  I've divided device-to-cloud messages into 2 categories:
* test messages, where we send at a regular rate and verify receipt.
* control messages, where we try to influence the behaviour of the service app.

Reply:
My point is, as a code reader, the word `test` is so ambiguous I have no idea what meaning you intend. I think it is poorly named.